src/ert/resources/shell_scripts/copy_directory.py

```diff
@@ -32,4 +32,6 @@ def copy_directory(src_path, target_path):
     try:
         copy_directory(src_path, target_path)
     except IOError as e:
-        sys.exit(f"COPY_DIRECTORY failed with the following error: {e}")
+        sys.exit(
+            f"COPY_DIRECTORY failed with the following error: {''.join(e.args[0])}"
```

Comment:
This is a little bit disturbing that we cannot do `{e}` on an IOError exception. Did you understand why this was needed?

Reply:
No, not other than I can guarantee that this comes directly from
```
shutil.copytree(src_path, target_path, dirs_exist_ok=True)
```
So this is some built-in weird behaviour it seems.

It is connected to this `dirs_exist_ok` argument, and when I change that to its default (`False`), the exception becomes more normal again; `FileExistsError(17, 'File exists')`

Reply:
👍🏻

Reply:
I think the problem may be upstream.

Reply:
Do we have tests that the other IOErrors are not mangled by this change?